src/Common/src/System/NonSerializedAttribute.cs

```diff
@@ -5,10 +5,10 @@
 namespace System
 {
     [AttributeUsage(AttributeTargets.Field, Inherited = false)]
-    public sealed class NonSerializedAttribute : Attribute
```

Comment:
Oh nvm I see what you did here, this is just the dummy attribute for non-netstandard1.7 builds right?

---
In reply to: [77096277](https://github.com/dotnet/corefx/pull/11312#discussion_r77096277) [](ancestors = 77096277)